sktime/regression/deep_learning/__init__.py

```diff
@@ -5,20 +5,18 @@
     "CNTCRegressor",
     "FCNRegressor",
     "InceptionTimeRegressor",
-    "LSTMFCNRegressor",
     "MACNNRegressor",
     "MCDCNNRegressor",
     "MLPRegressor",
     "ResNetRegressor",
     "SimpleRNNRegressor",
-    "TapNetRegressor",
+    "TapNetRegressor"
```

Comment:
please leave the comma, or you fail code formatting. Looks like AI action, please ensure you adhere to this guide:
https://www.sktime.net/en/stable/developer_guide/coding_standards.html